### PR TITLE
Add logo

### PR DESCRIPTION
**Pull request by sidraval**

### CODE REVIEW

**Comment by jakecraige** [comment]
What about shrinking the logo a bit? It's pretty huge when rendered: https://github.com/thoughtbot/Swish/blob/sr-add-logo/README.md

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Swish](https://raw.githubusercontent.com/thoughtbot/Swish/gh-pages/swish-logo-v4.jpg)]
```
Comment (jakecraige) [inline README.md]: You don't need these outer `[]` unless you're linking to something

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+<img src="https://raw.githubusercontent.com/thoughtbot/Swish/gh-pages/swish-logo-v4.jpg" width="500">
+
```
Comment (whmii) [inline README.md:2]: This line can be combined with the line below. something like…

```
# ![Swish](https://raw.githubusercontent.com/thoughtbot/Swish/gh-pages/swish-logo-v4.jpg) [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-brightgreen.svg?style=flat-square)](https://github.com/Carthage/Carthage)
```

Comment (gfontenot) [inline README.md:2]: We've been keeping these separate (see [Argo](https://github.com/thoughtbot/argo))

**Comment by gfontenot** [comment]
I don't mind the size of the logo, fwiw

**Comment by gfontenot** [comment]
But I'll defer to @whmii on the size. Once that's gotten a :+1: I think we can merge this.

**Comment by whmii** [comment]
yep, let me upload an svg instead of this jpg

**Pull request merged**